tests/WE2E/end_to_end_tests.sh

```diff
@@ -3,44 +3,55 @@
 #----------------------------------------------------------------------
 #  Automation of UFS Short Range Weather App Worfklow End to End Tests.
 #
-#  The script is dependant on a successful build of this repo. 
+#  The script is dependant on a successful build of this repo using the
+#  test/build.sh script in the ufs-srweather-app repository.
 #  The UFS build must be completed in a particular manner for this 
 #  script to function properly, notably the location of the build and
 #  bin directories: 
-#    BUILD_DIR=${APP_DIR}/build_${compiler}  
-#    BIN_DIR=${APP_DIR}/bin_${compiler}  
-#  
+#    BUILD_DIR=${APP_DIR}/build_${compiler}
+#    BIN_DIR=${APP_DIR}/bin_${compiler}
+#
 #  With the creating of a test_log directory and output files describing
 #  the result of each workflow status, a testing framework is established.
 #  The user must verify the contents of each experiment file in test_log/ 
-#  to check workflow success or failure.   
+#  to check workflow success or failure.
 #
-#  Example: . end_to_end_tests.sh hera zrtrr
+#  Example: ./end_to_end_tests.sh hera zrtrr
 #----------------------------------------------------------------------
 
 #-----------------------------------------------------------------------
 #  Set variables
 #-----------------------------------------------------------------------
 
-branches=( rrfs_ci ) 
+branches=( rrfs_ci )
+
+function usage {
+  echo
+  echo "Usage: $0 machine slurm_account  | -h"
+  echo
+  echo "       machine       [required] is one of: ${machines[@]}"
+  echo "       slurm_account [required] case sensitive name of the user-specific slurm account"
+  echo "       -h            display this help"
+  echo
+  exit 1
+
+}
+
+machines=( hera )
+
+if [ "$1" = "-h" ] ; then usage ; fi
+[[ $# -le 1 ]] && usage
 
 export machine=$1
-machine=`echo "${machine}" | tr '[A-Z]' '[a-z]'`  # scripts in sorc need lower case machine name
+machine=$(echo "${machine}" | tr '[A-Z]' '[a-z]')  # scripts in sorc need lower case machine name
```

Comment:
In general, it's easier to maintain and debug with the `$()` syntax than when using back ticks.

Reply:
I agree.